src/tests/strikethrough.rs

```diff
@@ -5,7 +5,7 @@ fn strikethrough() {
     html_opts!(
         [extension.strikethrough],
         concat!(
-            "This is ~strikethrough~.\n",
+            "This is ~~strikethrough~~.\n",
```

Comment:
This behaviour (when _only_ `strikethrough` is enabled, and not `subscript` too) should be unchanged, and so we should leave the test as-is.

Reply:
Will try to resolve strikethrough and subscript conflicts.